_dev/sources/miscBuilder.ts

```diff
@@ -24,22 +27,33 @@ export class MiscBuilder {
         Here: "HERE Maps",
         MapBox: "MapBox"
     };
+    private purgeSettings: any;
+    private emailSenderFrom: string;
+    private customerClassification: string;
 
     private abortCurrentTask () {
         this.currentTask && this.currentTask.abort && this.currentTask.abort();
         this.currentTask = null;
     }
 
+    //todo problem code...is this necessary?
     private getAllowedAddins (allAddins: string[]) {
```

Comment:
Just update this method to let "one line configuration" add-ins. This method filters out add-ins that are stored on the same server and can't be accessible from other servers

Reply:
The user requirement was to export all add-ins except the ConfigExport. I don't understand the relevance of same server. Can you please explain what you mean?

Reply:
There is a possibility to add an add-in by uploading its files to thi server via web interface:
![2020-03-09 07_53_33-Window](https://user-images.githubusercontent.com/25111354/76210844-53ce9100-61db-11ea-936a-ea4ba32d0d8b.png)
 We can't export add-ins that were created such way because we don't have correct URL to html file. So we should filter them out.

Reply:
ok yes agreed local file based addins should be ignored but the filter you had, removed other addins as well and that was not correct. It's the reason I took it out. It was removing addins that should not have been. On many tests addins that did not satisfy this rule were removed...I think there is a problem with the logic in this method. I will find you an example if I can.

Reply:
Yes. Just update this method to include "one line" add-ins

Reply:
There were other addins not included not only the "one line" add-ins...there was a problem with your method. I will try find an example...give a chance

Reply:
This database should have about 22 add-ins exported and I only get 13 with the original export software. https://my64.geotab.com/geotab_madrid. Could you take and look and let me know what your thoughts are on this?

Reply:
Just debug and fix this method

Reply:
done